app/interfaces/api/entities/search_result.rb

```diff
@@ -131,6 +132,10 @@ def cav_warning
       def supplementary
         object.case_type.eql?('Supplementary').to_i
       end
+
+      def lgfs_hardship
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3277/files#r412056455)

Reply:
I think this might be OK? I have called  the method def hardship in the agfs branch, so I don't think there should be a conflict (although possibly I should change this to def agfs_hardship)?

Reply:
see https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3282